modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -326,19 +327,31 @@ private CompletableFuture<?> processRequest(ReplicaRequest request, @Nullable Bo
         if (request instanceof ReadWriteSingleRowReplicaRequest) {
             var req = (ReadWriteSingleRowReplicaRequest) request;
 
-            return appendTxCommand(req.transactionId(), req.requestType(), () -> processSingleEntryAction(req));
+            return appendTxCommand(req.transactionId(), req.requestType(), req.full(), () -> processSingleEntryAction(req));
```

Comment:
With some slight updates in writeIntentResolution currently or if we will put committed value instead of writeIntent it'll be senseless  to acquire lock in case of single key operations.

Reply:
I think this is out of the scope of this PR.